src/neptune/new/__init__.py

```diff
@@ -24,6 +23,39 @@
 .. _Neptune docs page:
    https://docs.neptune.ai/api/neptune
 """
+__all__ = [
+    "types",
+    "ANONYMOUS",
+    "ANONYMOUS_API_TOKEN",
+    "NeptunePossibleLegacyUsageException",
+    "NeptuneUninitializedException",
+    "get_project",
+    "init",
```

Comment:
Are you sure that we want to keep such deprecated modules here as well (`init`)?

Reply:
Yes at this point :wink:. This could be limited as well after 1.0.0 release